build: Do not export `PKG_CONFIG_{PATH|LIBDIR}` variables

hebasto opened the pull request:
This is an alternative to bitcoin/bitcoin#25660 with no [drawbacks](https://github.com/bitcoin/bitcoin/pull/25660#issuecomment-1191281587).

Guix builds on `x86_64`:
```
8ac4c1164512d8aa1c4c3e379b5e12d7e91f196dc32decea422ab1edcb51461f  guix-build-b9f06bf05b67/output/aarch64-linux-gnu/SHA256SUMS.part
0e91431387030b7d2a6aba9368fab7fcf15931477b17c06350101bcb32a49217  guix-build-b9f06bf05b67/output/aarch64-linux-gnu/bitcoin-b9f06bf05b67-aarch64-linux-gnu-debug.tar.gz
d36ef4c9230d73d73760bf1533535aa8fd325584b11adb9101cff2097f548b88  guix-build-b9f06bf05b67/output/aarch64-linux-gnu/bitcoin-b9f06bf05b67-aarch64-linux-gnu.tar.gz
280d3c31e755b0e8e58cdcf184435fa6b7b69cf3446651ebfe76f9a632827094  guix-build-b9f06bf05b67/output/arm-linux-gnueabihf/SHA256SUMS.part
c6e7869ca390a8693c0d569ec89ffdcb128692e0e7cae89332adc0bd0663d0f3  guix-build-b9f06bf05b67/output/arm-linux-gnueabihf/bitcoin-b9f06bf05b67-arm-linux-gnueabihf-debug.tar.gz
2b0046e12d675c64a157265e16d014bd476be1c6f487f239cbdb151543790eb9  guix-build-b9f06bf05b67/output/arm-linux-gnueabihf/bitcoin-b9f06bf05b67-arm-linux-gnueabihf.tar.gz
92abf22c6c7e6a72d3018a836a2d3e16d2051af14a0c6add749eca268ddad470  guix-build-b9f06bf05b67/output/arm64-apple-darwin/SHA256SUMS.part
4cb47c5b5a302f0156ff0e998d0cb8103418f5e0f85b8d47d395771187cd8fda  guix-build-b9f06bf05b67/output/arm64-apple-darwin/bitcoin-b9f06bf05b67-arm64-apple-darwin-unsigned.dmg
660dab4a573b60a034f06f95a48563e9ea7d96632818140e578cd3ae972eb640  guix-build-b9f06bf05b67/output/arm64-apple-darwin/bitcoin-b9f06bf05b67-arm64-apple-darwin-unsigned.tar.gz
39ac1ecdce5a848aaca91f9f9dcc2a4436c1d257b27608191af45d4d29054990  guix-build-b9f06bf05b67/output/arm64-apple-darwin/bitcoin-b9f06bf05b67-arm64-apple-darwin.tar.gz
5afa45e1c9c2e31d97148e868415f6bbaf51def45aeaa32bb13b8a092284139e  guix-build-b9f06bf05b67/output/dist-archive/bitcoin-b9f06bf05b67.tar.gz
2aff14d389202d87266b93e1c17aa0ebbd9cde787349127f1a891dfddc41b675  guix-build-b9f06bf05b67/output/powerpc64-linux-gnu/SHA256SUMS.part
650c555c9d3d5b2ae18353d621b51cbdfbe5f2ebce31e7add47887adfd9b0283  guix-build-b9f06bf05b67/output/powerpc64-linux-gnu/bitcoin-b9f06bf05b67-powerpc64-linux-gnu-debug.tar.gz
38b33f13f2ac03ea2d864a02d5088b34441567bb6af04df9dc0c3aa4a9068cb2  guix-build-b9f06bf05b67/output/powerpc64-linux-gnu/bitcoin-b9f06bf05b67-powerpc64-linux-gnu.tar.gz
3b435cd35afe0990a6badb3d4f2a5e120c89644b566581db882241e82d5b94ca  guix-build-b9f06bf05b67/output/powerpc64le-linux-gnu/SHA256SUMS.part
3a3259a8c489e522a1763e4178f4d8b6b49cff927d5ebe9918c853f4d04547b7  guix-build-b9f06bf05b67/output/powerpc64le-linux-gnu/bitcoin-b9f06bf05b67-powerpc64le-linux-gnu-debug.tar.gz
006df723180d8260112e26089062f2a1ca4742099bf2d9455acd6be1b5395939  guix-build-b9f06bf05b67/output/powerpc64le-linux-gnu/bitcoin-b9f06bf05b67-powerpc64le-linux-gnu.tar.gz
d13f9c8e9396c46496e06cf6bfbaffae3980e6305024a1e447f73346e66e48a5  guix-build-b9f06bf05b67/output/riscv64-linux-gnu/SHA256SUMS.part
9b18dfafd51a6d249ed74d884c4a8d1b2cf320133cea8008742bc93583cff19e  guix-build-b9f06bf05b67/output/riscv64-linux-gnu/bitcoin-b9f06bf05b67-riscv64-linux-gnu-debug.tar.gz
76d29d553f06c7098c67e8fc95f83c45619860988668567f37946efd63668cef  guix-build-b9f06bf05b67/output/riscv64-linux-gnu/bitcoin-b9f06bf05b67-riscv64-linux-gnu.tar.gz
6661426b6180c8bb908b05f1ea4e8fe81acc02a443784e0ca042feebb1c8770c  guix-build-b9f06bf05b67/output/x86_64-apple-darwin/SHA256SUMS.part
8310942fddcbf991d3162f94b9e0f2f9f413f10089b6ac31d5c3a73039c3e987  guix-build-b9f06bf05b67/output/x86_64-apple-darwin/bitcoin-b9f06bf05b67-x86_64-apple-darwin-unsigned.dmg
cdcbb08d7596a3f9a0b3816b113e7e4afd435fa82ae20d2d6750e30ccb13d820  guix-build-b9f06bf05b67/output/x86_64-apple-darwin/bitcoin-b9f06bf05b67-x86_64-apple-darwin-unsigned.tar.gz
ac02894454dcee5c822304ab83165e500a882f7b5dd4d5f3645ac526652eb707  guix-build-b9f06bf05b67/output/x86_64-apple-darwin/bitcoin-b9f06bf05b67-x86_64-apple-darwin.tar.gz
fa55686ae7c977ee9ec0213cad8f4021e81153a6de60a5b9f74fb840f173fdfa  guix-build-b9f06bf05b67/output/x86_64-linux-gnu/SHA256SUMS.part
cf40ec54ea736876a0fa5060ecad41d9215762b6d9b89fb2716cf073729097b9  guix-build-b9f06bf05b67/output/x86_64-linux-gnu/bitcoin-b9f06bf05b67-x86_64-linux-gnu-debug.tar.gz
4479fe5dc29925d7b51ed20faa44a87b68aa40b1fef979f1061240325892373f  guix-build-b9f06bf05b67/output/x86_64-linux-gnu/bitcoin-b9f06bf05b67-x86_64-linux-gnu.tar.gz
bef988880e6dbb7be90c4b2b56d5d9a68b91dceb64b2fa38e4d67e8c8cc5a78a  guix-build-b9f06bf05b67/output/x86_64-w64-mingw32/SHA256SUMS.part
0040f79968d8ebb507358ee86797880a019f9730b92576af125b778bcf5ee233  guix-build-b9f06bf05b67/output/x86_64-w64-mingw32/bitcoin-b9f06bf05b67-win64-debug.zip
ac63bf2dbf78361133043db7fa24be51c25fc5ddbbe19ea4a1c78e0843054757  guix-build-b9f06bf05b67/output/x86_64-w64-mingw32/bitcoin-b9f06bf05b67-win64-setup-unsigned.exe
0d9e317a95a613eb2e9216c4c9f5b0046ff52e3b11af80b8de9ac89209f33ab7  guix-build-b9f06bf05b67/output/x86_64-w64-mingw32/bitcoin-b9f06bf05b67-win64-unsigned.tar.gz
1a47e56d06207f3c86310c6eaec66f2c7693ca810de27ab2f97e67086239d396  guix-build-b9f06bf05b67/output/x86_64-w64-mingw32/bitcoin-b9f06bf05b67-win64.zip
```

hebasto committed:
build: Do not export `PKG_CONFIG_{PATH|LIBDIR}` variables

real-or-random commented on configure.ac:
```diff
@@ -19,6 +19,16 @@ if test "$PKG_CONFIG" = ""; then
   AC_MSG_ERROR([pkg-config not found])
 fi
 
+if test -n "$PKG_CONFIG_PATH"; then
+  PKG_CONFIG="PKG_CONFIG_PATH=$PKG_CONFIG_PATH $PKG_CONFIG"
+fi
+if test -n "$PKG_CONFIG_LIBDIR"; then
+  PKG_CONFIG="PKG_CONFIG_LIBDIR=$PKG_CONFIG_LIBDIR $PKG_CONFIG"
+fi
+if test -n "$PKG_CONFIG_PATH" || test -n "$PKG_CONFIG_LIBDIR"; then
+  PKG_CONFIG="env $PKG_CONFIG"
+fi
```
nit: Maybe this is nicer (shorter, less coupled):

```suggestion
if test -n "$PKG_CONFIG_PATH"; then
  PKG_CONFIG="env PKG_CONFIG_PATH=$PKG_CONFIG_PATH $PKG_CONFIG"
fi
if test -n "$PKG_CONFIG_LIBDIR"; then
  PKG_CONFIG="env PKG_CONFIG_LIBDIR=$PKG_CONFIG_LIBDIR $PKG_CONFIG"
fi
```

This may create two `env` invocations but it still works.

real-or-random replied:
one-liner, not sure if it's better ;)
```
PKG_CONFIG="env ${PKG_CONFIG_PATH+PKG_CONFIG_PATH=}${PKG_CONFIG_PATH} ${PKG_CONFIG_LIBDIR+PKG_CONFIG_LIBDIR=}${PKG_CONFIG_LIBDIR} $PKG_CONFIG"
```

fanquake replied:
I don't really mind which of the 3 versions we pick. The shorter less-coupled version seems fine.

hebasto replied:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/25687#issuecomment-1196481084).

real-or-random reviewed (approved):
Oh nice, using `env` is much better than the two ideas I suggested in #25660.

utACK a19aaf1f476177936054fedf2bf396ff310bf459

jarolrod commented:
GUIX hashes

x86:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum

08edb79494b3cb4453d6df5b22b44ebc6e9ab6c301e834e4992964ecfa06ab6c  guix-build-a19aaf1f4761/output/aarch64-linux-gnu/SHA256SUMS.part
e318e0771457927902a94dc5cb82f67753b3879d3db0e00b679ce17611092e9d  guix-build-a19aaf1f4761/output/aarch64-linux-gnu/bitcoin-a19aaf1f4761-aarch64-linux-gnu-debug.tar.gz
2efdd73d4d95168888243c1b31649d922d8ee9a5de55717516dc173e5615cbca  guix-build-a19aaf1f4761/output/aarch64-linux-gnu/bitcoin-a19aaf1f4761-aarch64-linux-gnu.tar.gz
c7deba594bf5b48197e86bcd739908620ab5e194f7e98ecce59245064c2f8b56  guix-build-a19aaf1f4761/output/arm-linux-gnueabihf/SHA256SUMS.part
951e8792553103f8a4056646fee708991f55de9f240db89a756bd9daf008b999  guix-build-a19aaf1f4761/output/arm-linux-gnueabihf/bitcoin-a19aaf1f4761-arm-linux-gnueabihf-debug.tar.gz
7e765c9ee58099990416f9f64919586645306293d1fbd81ff753bb103bff8e1a  guix-build-a19aaf1f4761/output/arm-linux-gnueabihf/bitcoin-a19aaf1f4761-arm-linux-gnueabihf.tar.gz
ec80d11c55f015f79fb458d54625a5cc428964036c07df2fc898c57b7695b3be  guix-build-a19aaf1f4761/output/arm64-apple-darwin/SHA256SUMS.part
57b22a9549d6e1f37be7283c949c2d6745d5bac5aa931108f3ad0fa3c3f0f421  guix-build-a19aaf1f4761/output/arm64-apple-darwin/bitcoin-a19aaf1f4761-arm64-apple-darwin-unsigned.dmg
681a71276ff67b70ca413a7e5a1d07fb92502297130db7ea2326f03164159f80  guix-build-a19aaf1f4761/output/arm64-apple-darwin/bitcoin-a19aaf1f4761-arm64-apple-darwin-unsigned.tar.gz
343852ebedc130097b1e5e67d2c46d0d09444fab450c6be1f40d0612217fa83b  guix-build-a19aaf1f4761/output/arm64-apple-darwin/bitcoin-a19aaf1f4761-arm64-apple-darwin.tar.gz
59fca1e0261c4a22ee67f8eee3c0351f7623b931461742bd20d009d8cfc4aa66  guix-build-a19aaf1f4761/output/dist-archive/bitcoin-a19aaf1f4761.tar.gz
30315d1453a8dca9a9864d8ee5360cca750d685bc42283c4e098c1998568d5cd  guix-build-a19aaf1f4761/output/powerpc64-linux-gnu/SHA256SUMS.part
9858a98b63fc58284ca3a80be50882bc72b4b946d42aca7893c5de2d97c0695a  guix-build-a19aaf1f4761/output/powerpc64-linux-gnu/bitcoin-a19aaf1f4761-powerpc64-linux-gnu-debug.tar.gz
6986bc5d5d81246173db28ae69328bc166468713fa3ed9cbdfef8754717f8276  guix-build-a19aaf1f4761/output/powerpc64-linux-gnu/bitcoin-a19aaf1f4761-powerpc64-linux-gnu.tar.gz
0aef99de023cb7c8bd73bc4bbd0d283c8b1ea65d6a1d81aafdf600662628df8c  guix-build-a19aaf1f4761/output/powerpc64le-linux-gnu/SHA256SUMS.part
c86062f95ef7e0e88c76e1d50632b70f79a07e6408004263b8c5a4377bdb057a  guix-build-a19aaf1f4761/output/powerpc64le-linux-gnu/bitcoin-a19aaf1f4761-powerpc64le-linux-gnu-debug.tar.gz
6afc13af119e805ec279fd46bc0440405ad04d0488e18c55914ea9d8116f388e  guix-build-a19aaf1f4761/output/powerpc64le-linux-gnu/bitcoin-a19aaf1f4761-powerpc64le-linux-gnu.tar.gz
4ceffd1daca58f065f458c382be859696855a23b95dd8b627977a1099297450d  guix-build-a19aaf1f4761/output/riscv64-linux-gnu/SHA256SUMS.part
9f55cab8332193fd2b22908d7a20e3c9e2ef4b8419a95bd19226a911c72e3c58  guix-build-a19aaf1f4761/output/riscv64-linux-gnu/bitcoin-a19aaf1f4761-riscv64-linux-gnu-debug.tar.gz
d10af8de9dcfc6131ff08a83d40f1a75bec549be5ca35bebc75ade4fee5ecf81  guix-build-a19aaf1f4761/output/riscv64-linux-gnu/bitcoin-a19aaf1f4761-riscv64-linux-gnu.tar.gz
980b3286d347bfbc7e7097c58c6f171099fb1f5bee686624833e097a113f0015  guix-build-a19aaf1f4761/output/x86_64-apple-darwin/SHA256SUMS.part
d37a7121692bfdb5a2ee80be78b9f287f0b4e3be3279bcb23aff3e541da457b0  guix-build-a19aaf1f4761/output/x86_64-apple-darwin/bitcoin-a19aaf1f4761-x86_64-apple-darwin-unsigned.dmg
1d8a784306de50df2dffc3d387a426bf9c27e82014e4a2752737413270483988  guix-build-a19aaf1f4761/output/x86_64-apple-darwin/bitcoin-a19aaf1f4761-x86_64-apple-darwin-unsigned.tar.gz
5e78d76c04687d5deff3bf112fdd18223ee72f7b1be5468de5010a52438d6baa  guix-build-a19aaf1f4761/output/x86_64-apple-darwin/bitcoin-a19aaf1f4761-x86_64-apple-darwin.tar.gz
438c0920cfaaadea2842dac1f17b39c0a0f7acc8a68ec6cb6ce987f314dbe875  guix-build-a19aaf1f4761/output/x86_64-linux-gnu/SHA256SUMS.part
f15b8101e9bc2ad3cd869f6a9c65ac531550732591c4bfdd37ae7f826b3d23e7  guix-build-a19aaf1f4761/output/x86_64-linux-gnu/bitcoin-a19aaf1f4761-x86_64-linux-gnu-debug.tar.gz
cdd3e762619661a358ebedcdf9f3acdf5504dff4fb24ce6216543ee201b521f5  guix-build-a19aaf1f4761/output/x86_64-linux-gnu/bitcoin-a19aaf1f4761-x86_64-linux-gnu.tar.gz
af564e9f2769c7c9b828d531985814d608ff0e5e1489614ca3c72c2bac967cdd  guix-build-a19aaf1f4761/output/x86_64-w64-mingw32/SHA256SUMS.part
4c7d3fedd895358270feaade83f9a350b4ce8e0f63fc0cee9dcdb92867ffe041  guix-build-a19aaf1f4761/output/x86_64-w64-mingw32/bitcoin-a19aaf1f4761-win64-debug.zip
03f1db5eb5c4f4a2586691782eae3f20f997092016264d6d7fc75105b0e9bb3c  guix-build-a19aaf1f4761/output/x86_64-w64-mingw32/bitcoin-a19aaf1f4761-win64-setup-unsigned.exe
e106534f3cb8714fcfa4e6cc46e61a76645863ba8c28905f417a5fc525f8f3fd  guix-build-a19aaf1f4761/output/x86_64-w64-mingw32/bitcoin-a19aaf1f4761-win64-unsigned.tar.gz
df401a57d5a92475aa959a93ad5cc366ef18d2f22b04bf4416b9ce1ba7707175  guix-build-a19aaf1f4761/output/x86_64-w64-mingw32/bitcoin-a19aaf1f4761-win64.zip
```

arm64:
```
$ env HOSTS='arm-linux-gnueabihf arm64-apple-darwin powerpc64-linux-gnu powerpc64le-linux-gnu riscv64-linux-gnu x86_64-apple-darwin x86_64-linux-gnu x86_64-w64-mingw32' ./contrib/guix/guix-build
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum

c7deba594bf5b48197e86bcd739908620ab5e194f7e98ecce59245064c2f8b56  guix-build-a19aaf1f4761/output/arm-linux-gnueabihf/SHA256SUMS.part
951e8792553103f8a4056646fee708991f55de9f240db89a756bd9daf008b999  guix-build-a19aaf1f4761/output/arm-linux-gnueabihf/bitcoin-a19aaf1f4761-arm-linux-gnueabihf-debug.tar.gz
7e765c9ee58099990416f9f64919586645306293d1fbd81ff753bb103bff8e1a  guix-build-a19aaf1f4761/output/arm-linux-gnueabihf/bitcoin-a19aaf1f4761-arm-linux-gnueabihf.tar.gz
68887d3df73b16857093eda632a14a89d9efe71a3b462d1b533593d025e62550  guix-build-a19aaf1f4761/output/arm64-apple-darwin/SHA256SUMS.part
804dbef3f7f5b49dee2687606958f6f08e13f519fe3c7c8dc1d75c2ee677f66f  guix-build-a19aaf1f4761/output/arm64-apple-darwin/bitcoin-a19aaf1f4761-arm64-apple-darwin-unsigned.dmg
e88a7ec23c040cea105a5201570a71d89b3ccacaf3769d1b711a646ebc957e01  guix-build-a19aaf1f4761/output/arm64-apple-darwin/bitcoin-a19aaf1f4761-arm64-apple-darwin-unsigned.tar.gz
670077cc6edaa9aa9460e31b0d2b9636a1924cd65f555979014fa3f77ec9100e  guix-build-a19aaf1f4761/output/arm64-apple-darwin/bitcoin-a19aaf1f4761-arm64-apple-darwin.tar.gz
59fca1e0261c4a22ee67f8eee3c0351f7623b931461742bd20d009d8cfc4aa66  guix-build-a19aaf1f4761/output/dist-archive/bitcoin-a19aaf1f4761.tar.gz
30315d1453a8dca9a9864d8ee5360cca750d685bc42283c4e098c1998568d5cd  guix-build-a19aaf1f4761/output/powerpc64-linux-gnu/SHA256SUMS.part
9858a98b63fc58284ca3a80be50882bc72b4b946d42aca7893c5de2d97c0695a  guix-build-a19aaf1f4761/output/powerpc64-linux-gnu/bitcoin-a19aaf1f4761-powerpc64-linux-gnu-debug.tar.gz
6986bc5d5d81246173db28ae69328bc166468713fa3ed9cbdfef8754717f8276  guix-build-a19aaf1f4761/output/powerpc64-linux-gnu/bitcoin-a19aaf1f4761-powerpc64-linux-gnu.tar.gz
0aef99de023cb7c8bd73bc4bbd0d283c8b1ea65d6a1d81aafdf600662628df8c  guix-build-a19aaf1f4761/output/powerpc64le-linux-gnu/SHA256SUMS.part
c86062f95ef7e0e88c76e1d50632b70f79a07e6408004263b8c5a4377bdb057a  guix-build-a19aaf1f4761/output/powerpc64le-linux-gnu/bitcoin-a19aaf1f4761-powerpc64le-linux-gnu-debug.tar.gz
6afc13af119e805ec279fd46bc0440405ad04d0488e18c55914ea9d8116f388e  guix-build-a19aaf1f4761/output/powerpc64le-linux-gnu/bitcoin-a19aaf1f4761-powerpc64le-linux-gnu.tar.gz
4ceffd1daca58f065f458c382be859696855a23b95dd8b627977a1099297450d  guix-build-a19aaf1f4761/output/riscv64-linux-gnu/SHA256SUMS.part
9f55cab8332193fd2b22908d7a20e3c9e2ef4b8419a95bd19226a911c72e3c58  guix-build-a19aaf1f4761/output/riscv64-linux-gnu/bitcoin-a19aaf1f4761-riscv64-linux-gnu-debug.tar.gz
d10af8de9dcfc6131ff08a83d40f1a75bec549be5ca35bebc75ade4fee5ecf81  guix-build-a19aaf1f4761/output/riscv64-linux-gnu/bitcoin-a19aaf1f4761-riscv64-linux-gnu.tar.gz
980b3286d347bfbc7e7097c58c6f171099fb1f5bee686624833e097a113f0015  guix-build-a19aaf1f4761/output/x86_64-apple-darwin/SHA256SUMS.part
d37a7121692bfdb5a2ee80be78b9f287f0b4e3be3279bcb23aff3e541da457b0  guix-build-a19aaf1f4761/output/x86_64-apple-darwin/bitcoin-a19aaf1f4761-x86_64-apple-darwin-unsigned.dmg
1d8a784306de50df2dffc3d387a426bf9c27e82014e4a2752737413270483988  guix-build-a19aaf1f4761/output/x86_64-apple-darwin/bitcoin-a19aaf1f4761-x86_64-apple-darwin-unsigned.tar.gz
5e78d76c04687d5deff3bf112fdd18223ee72f7b1be5468de5010a52438d6baa  guix-build-a19aaf1f4761/output/x86_64-apple-darwin/bitcoin-a19aaf1f4761-x86_64-apple-darwin.tar.gz
438c0920cfaaadea2842dac1f17b39c0a0f7acc8a68ec6cb6ce987f314dbe875  guix-build-a19aaf1f4761/output/x86_64-linux-gnu/SHA256SUMS.part
f15b8101e9bc2ad3cd869f6a9c65ac531550732591c4bfdd37ae7f826b3d23e7  guix-build-a19aaf1f4761/output/x86_64-linux-gnu/bitcoin-a19aaf1f4761-x86_64-linux-gnu-debug.tar.gz
cdd3e762619661a358ebedcdf9f3acdf5504dff4fb24ce6216543ee201b521f5  guix-build-a19aaf1f4761/output/x86_64-linux-gnu/bitcoin-a19aaf1f4761-x86_64-linux-gnu.tar.gz
af564e9f2769c7c9b828d531985814d608ff0e5e1489614ca3c72c2bac967cdd  guix-build-a19aaf1f4761/output/x86_64-w64-mingw32/SHA256SUMS.part
4c7d3fedd895358270feaade83f9a350b4ce8e0f63fc0cee9dcdb92867ffe041  guix-build-a19aaf1f4761/output/x86_64-w64-mingw32/bitcoin-a19aaf1f4761-win64-debug.zip
03f1db5eb5c4f4a2586691782eae3f20f997092016264d6d7fc75105b0e9bb3c  guix-build-a19aaf1f4761/output/x86_64-w64-mingw32/bitcoin-a19aaf1f4761-win64-setup-unsigned.exe
e106534f3cb8714fcfa4e6cc46e61a76645863ba8c28905f417a5fc525f8f3fd  guix-build-a19aaf1f4761/output/x86_64-w64-mingw32/bitcoin-a19aaf1f4761-win64-unsigned.tar.gz
df401a57d5a92475aa959a93ad5cc366ef18d2f22b04bf4416b9ce1ba7707175  guix-build-a19aaf1f4761/output/x86_64-w64-mingw32/bitcoin-a19aaf1f4761-win64.zip
```

fanquake commented on configure.ac:
```diff
@@ -19,6 +19,16 @@ if test "$PKG_CONFIG" = ""; then
   AC_MSG_ERROR([pkg-config not found])
 fi
 
+if test -n "$PKG_CONFIG_PATH"; then
```
I think we could add a note here.

hebasto replied:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/25687#issuecomment-1196481084).

hebasto commented:
Updated a19aaf1f476177936054fedf2bf396ff310bf459 -> b9f06bf05b67239cea80998a747f73a2ed3e7be6 ([pr25687.01](https://github.com/hebasto/bitcoin/commits/pr25687.01) -> [pr25687.02](https://github.com/hebasto/bitcoin/commits/pr25687.02), [diff](https://github.com/hebasto/bitcoin/compare/pr25687.01..pr25687.02)):

- addressed @real-or-random's [comment](https://github.com/bitcoin/bitcoin/pull/25687#discussion_r928218491)
- addressed @fanquake's [comment](https://github.com/bitcoin/bitcoin/pull/25687#discussion_r929808804)

real-or-random reviewed (approved):
utACK b9f06bf05b67239cea80998a747f73a2ed3e7be6
